spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -263,56 +252,44 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
 
       links = document.createElement('div')
       links.innerHTML = '<div class="fully-structured-download-links">' +
-            '<a href="https://assets.publishing.service.gov.uk/one.pdf" external="true">PDF</a>' +
-            '<a href="https://assets.publishing.service.gov.uk/two.xslt" external="true">Spreadsheet</a>' +
-            '<a href="https://www.gov.uk/government/uploads/system/three.doc" external="false">Document</a>' +
+            '<a href="https://assets.publishing.service.gov.uk/one.pdf" external="true" link-domain="https://assets.publishing.service.gov.uk">PDF</a>' +
```

Comment:
Nitpick: you've used `link-domain` here but `link_domain` previously, might be good to be consistent.